Library/Tracking/Koala.swift

```diff
@@ -2127,39 +2135,6 @@ private func projectProperties(
     .prefixedKeys(prefix)
 }
 
-private func properties(
```

Comment:
This was almost completely redundant with `projectProperties`. Any actually unique props were moved up to the actual tracking function.